src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1194,6 +1202,8 @@ private ExtBidResponse toExtBidResponse(List<BidderResponse> results, BidRequest
 
         boolean shouldCacheBids;
 
+        boolean shouldCacheOnlyWinningBids;
```

Comment:
Do we really need this field?

Reply:
We are analysing current flags in order to decide what we need.

Reply:
i think you're using `org.prebid.server.auction.ExchangeService#shouldCacheOnlyWinningBids` for that.
The `org.prebid.server.auction.ExchangeService.BidRequestCacheInfo` model is rather request specific then configuration. So, it can be utilized if the `request.ext.prebid.cache.onlyWinningBids` will be added.